src/ocean/task/util/StreamProcessor.d

```diff
@@ -184,6 +184,7 @@ class StreamProcessor ( TaskT : Task )
 
         if (throttler_config.resume_point == size_t.max)
             throttler_config.resume_point = total / 5;
+		else
```

Comment:
Yep, for same reason. I must have been distracted with something else before fully changing the code accordingly which is why it looks so weird now.